src/index/blockfilterindex.cpp

```diff
@@ -326,7 +327,7 @@ static bool LookupOne(const CDBWrapper& db, const CBlockIndex* block_index, DBVa
 }
 
 static bool LookupRange(CDBWrapper& db, const std::string& index_name, int start_height,
-                        const CBlockIndex* stop_index, std::vector<DBVal>& results)
+                        const CBlockIndex* stop_index, std::vector<std::pair<uint256, DBVal>>& results)
```

Comment:
Wouldn't hurt to add a comment for this. By reading only the function's signature, it's not clear what the uin256 is.